### PR TITLE
Add syntax diagram for CREATE SOURCE

### PR DESCRIPTION
**Pull request by bernscode**
Add syntax diagram for CREATE SOURCE.

## Info
- **Description**: 
Add syntax diagram for CREATE SOURCE.

- **Preview**: 
[ Paste the preview link to the edited page here. ]

- **Related code PR**: 
N/A

- **Related doc issue**: 
Resolves no doc issue.

- **Notes**: 
[ Any additional information? ]

## Before merging
  - [ ] (For version-specific PR) I have selected the corresponding software version in **Milestone** and linked the related doc issue to this PR in **Development**.
  - [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`bernscode`, `CharlieSYH`, `emile-00`, & `hengm3467`). 
  - [ ] I have checked the doc site preview, and the updated parts look good. <details><summary>How?</summary>Scroll down and open this link: image
</details>

### CODE REVIEW

**Comment by hengm3467** [comment]
@bernscode I just realized that the code syntax needs to be updated. Let me update the code syntax first, and then you can update the diagram syntax later.

**Comment by hengm3467** [comment]
@bernscode I've updated the code syntax, you can update the diagram syntax now. Thanks.

**Comment by bernscode** [comment]
@hengm3467 Great! I've made the update in the latest commit. Thanks!

**Review by hengm3467 — APPROVED** [review]
Good work! Thanks. @bernscode

**Comment by hengm3467** [comment]
@bernscode  I just realized that maybe it's better if we also update the all integration guides in "Data ingestion", as they are similar and it's easier to add diagrams while the memory is still fresh. Could you please add syntax diagrams to applicable pages under "Data ingestion"? Thanks. 

It's up to you to include the changes in this PR or start a new PR.

image

**Comment by bernscode** [comment]
@hengm3467 Thanks! Yeah, that sounds good. I'll work on those and add them to this PR.

File: docs/create-source/create-source-kinesis.md [hunk 1]
```diff
@@ -23,6 +23,72 @@ ROW FORMAT data_format
...
+            rr.Sequence(
+                rr.Terminal('MESSAGE'),
+                rr.NonTerminal('message', 'skip'),
+            ),
+        ),
+        rr.Optional(
+            rr.Sequence(
+                rr.Terminal('ROW SCHEMA LOCATION'),
+                rr.Terminal('location'),
+                rr.Terminal(';'),
```
Comment (hengm3467) [inline docs/create-source/create-source-kinesis.md]: This occurrence of ";" is not necessary, as we already have one ending ; that follows. @bernscode

**Pull request merged**